docs/tests/integration/post_v1_message-batches/happy_path.md

```diff
@@ -18,6 +18,21 @@
 - Response contains correctly formatted link to new message URI
 
 
+### Scenario: An API consumer creating a batch of messages with valid contact details receives a 201 response
+
+**Given** the API consumer provides valid contact details for a recipient in their new message batch
+<br/>
+**When** the request is submitted
+<br/>
+**Then** the response is a 201 success
+<br/>
+
+**Asserts**
+- Response returns a 201 status code
+- Response body matches expected result
+- Response contains request ID
```

Comment:
This fix should be applied to all files, but as its out of scope for this PR i've raised https://nhsd-jira.digital.nhs.uk/browse/CCM-6748 to deal with this issue